tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -109,4 +126,85 @@ public function test_get_post_type_icon() {
 		$valid            = $this->call_private_method( $post_type_object, 'get_post_type_icon' );
 		$this->assertContains( 'data:image/svg+xml;base64', $valid );
 	}
+
+	/**
+	 * @covers \Google\Web_Stories\Story_Post_Type::admin_enqueue_scripts
+	 */
+	public function test_admin_enqueue_scripts() {
+		$post_type_object = new \Google\Web_Stories\Story_Post_Type();
+		set_current_screen( 'post.php' );
+		get_current_screen()->post_type = \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG;
+		get_current_screen()->base      = 'post';
```

Comment:
Same optimization as in #2849:

```suggestion
		$GLOBALS['current_screen'] = convert_to_screen( \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG );
```